core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -0,0 +1,84 @@
+<script type="text/ng-template" id="simpleEditor/body">
+  <div class="main">
+    <div class="content">
+      <h2>What would you like to teach?</h2>
+      <editor-plaintext-field original-value="explorationTitleService.savedMemento"
+                              on-finish-editing="saveTitle(newValue)"
+                              placeholder="Type the title of your exploration.">
+      </editor-plaintext-field>
+
+      <h2>Introduction</h2>
+      <editor-html-field original-value="initialState.content[0].value"
+                         on-finish-editing="saveIntroduction(newValue)">
+      </editor-html-field>
+
+      <div ng-repeat="question in questions track by $index">
+        <h2>Question <[$index + 1]></h2>
+
+        <multiple-choice-editor ng-if="question.interactionId === 'MultipleChoiceInput'"
```

Comment:
No -- this is hard, and I considered that too. I think this way is simpler. The switch will need to happen somewhere.